byu-calendar-tile/byu-calendar-tile.scss

```diff
@@ -110,36 +113,43 @@ $medgray: #767676;
     }
     
 	#month-name {
-		padding: 10px;
+		margin: 10px;
 		font-size: 22px;
 		text-transform: uppercase;
-		margin-bottom: 0px;
+		height: 30px;
     }
     
 	#day-number {
 		font-size: 60px;
 		margin-bottom: 10px;
 		font-weight: bold;
+		height: 68px;
+		text-align: center;
+		line-height: 68px;
     }
     
 	#title ::slotted(*) {
 		color: $navy!important;
 		font-size: 18px;
 		font-weight: bold;
 		text-align: center;
+		overflow: hidden;
+		text-overflow: ellipsis;
 		padding: 5px 0px 10px;
 		height: 120px;
 		width: 150px;
 	}
 	#time ::slotted(*) {
 		font-size: 14px;
 		padding: 8px 0px!important;
+		height: 30px;
```

Comment:
When you remove the height in firefox, the location name slips outside of the card itself like this:

![image](https://user-images.githubusercontent.com/37681688/41985717-5dbd3d2a-79f1-11e8-9b29-d8de38d4c525.png)

However, you can change the padding, and it will look pretty normal.

Reply:
when you say change the padding, what do you mean?
what we should do instead is we should set a height:auto so that it gets taller when there are 2 lines of location text. Can you try that and test for this long location case and a short location case?

Reply:
What I mean by changing the padding is by changing it to `padding: 8px 0 0 !important;`